### PR TITLE
Bump font-awesome-sass from 5.15.1 to 6.4.0

### PR DESCRIPTION
**Pull request by karim-semmoud**
## Reference
https://fontawesome.com/docs/web/use-with/ruby-on-rails
https://fontawesome.com/docs/web/setup/upgrade/
Relates to #4974 

## Objectives

This pull request aims at upgrading Font Awesome from 5.15.1 to 6.4.0
-Update the gem version
-Update SCSS import
-Upload 6.4.0 SVG icons
-Rename depreciated icons name

### CODE REVIEW

**Comment by javierm** [comment]
Hi, @karim-semmoud :smile:.

Thanks a lot for this pull request! We got stuck when trying to do the upgrade last year and postponed it, and this pull request helps a lot.

We've got a few things on our schedule and aren't sure when we'll be able to review this pull request. Sorry for that! In the meantime, could you provide the link where you downloaded the icons from?

Thanks!

**Comment by karim-semmoud** [comment]
Hi @javierm 
There is no rush for it as it is not essential and the version 6 doesn't bring much more, except new icons mainly

I downloaded the full SVG set for the 6.4.0 For The Web (Free)
[https://fontawesome.com/download](https://fontawesome.com/download)
[https://use.fontawesome.com/releases/v6.4.0/fontawesome-free-6.4.0-web.zip](https://use.fontawesome.com/releases/v6.4.0/fontawesome-free-6.4.0-web.zip)

Let me know if you need me for anything

**Review by javierm — COMMENTED** [review]
@karim-semmoud This is really, really, really great. You can't imagine how helpful this is. Than you so much! :pray:

I've left a couple of suggestions. Let me know what you think!

File: Gemfile [hunk 1]
```diff
@@ -21,7 +21,8 @@ gem "delayed_job_active_record", "~> 4.1.7"
 gem "devise", "~> 4.8.1"
 gem "devise-security", "~> 0.17.0"
 gem "file_validators", "~> 3.0.0"
-gem "font-awesome-sass", "~> 5.15.1" # Remember to update vendor/assets/images/fontawesome when updating this gem
+gem "font-awesome-sass", "~> 6.4.0"
+# Remember to update vendor/assets/images/fontawesome when updating this gem
```
Comment (javierm) [inline Gemfile:25]: Sorry, I forgot one thing! :pray:

Could you remove the commit changing this line? You can ignore the warning in this case.

The thing is, it's much easier for developers like me to accidentally ignore the comment if it's on a different line. If it's on the same line, even I'm going to notice it when upgrading (or, at least, I hope I will! :crossed_fingers:).

**Comment by javierm** [comment]
Hi, @karim-semmoud :smile:.

Are you still working on this pull request? :thinking: If you aren't, that's fine :ok_hand:; we'll open a new pull request and will keep you as the author of the commits.

**Comment by karim-semmoud** [comment]
Hi @javierm I am not working on this anymore but feel free to use the previous commits

**Comment by javierm** [comment]
@karim-semmoud Thank you once again for everything you've done in this pull request :pray:.

Closing in favor of #5449.

**Pull request closed without merging**